core/unit_test/TestTeamReductionScan.hpp

```diff
@@ -48,6 +48,8 @@
 
 namespace Test {
 
+// FIXME_SYCL needs team_scan
+#ifndef KOKKOS_ENABLE_SYCL
```

Comment:
Does this work with #3815?

Reply:
Well, I need both pull requests for that. :slightly_smiling_face:

Reply:
I'll look into it but consider it part of #3815 so I would appreciate if we can move forward here independently.

Reply:
This works now in #3815 after rebasing there.